### DISCUSSION TITLE
Donjons et dragons

### DISCUSSION THREAD

**Discussion by evefevrier** [integration2/quizTIM2023#55] [Q&A]
Commentaires sur ce quiz.

**Comment by raphaellabbelatreille** [comment]
J'adore le style de la page.  La typographie est simple mais tout les encadrement et les image sont fidèles à ce que l'on verait sur une fiche de personnage de Donjons et dragon. Les questions sont simple et plaisant à répondre.
La seule chose que j'aurais à reprendre est l'animation sur les choix de réponses. Le fait que la case se déplace vers le haut-gauche me semble inaproprié. Du genre, la page ressemble à une fiche papier et le bouton pourait tout simplement rétrécir pour montré le :hover.
 -Raphaël Labbé-Latreille

**Comment by evefevrier** [comment]
@Heyliya 
J'aime bien la sobriété esthétique de l'interface qui cache bien sa complexité.
Intéressante incursion dans l'univers D&D,
